New log file opened at Fri Dec 18 17:37:28 CET 2015 by 7436@PC-14874
Taking next connection to: 49321
New client attempting to connect on port: 49322-49321
  Address = /139.63.20.119:49321
This client on port 49322 has uuid 8aa67c91-fdc7-4c65-a51a-bba36c82fd85 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-8aa67c91-fdc7-4c65-a51a-bba36c82fd85
Taking next connection to: 49321
New client attempting to connect on port: 49323-49321
  Address = /139.63.20.119:49321
Handshake exception on socket: 49323
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49321
New client attempting to connect on port: 49324-49321
  Address = /139.63.20.119:49321
This client on port 49324 has uuid 5399a23d-5ec0-417e-bc05-4cfb0f566cd4 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-8aa67c91-fdc7-4c65-a51a-bba36c82fd85, activator-5399a23d-5ec0-417e-bc05-4cfb0f566cd4
Taking next connection to: 49321
New client attempting to connect on port: 49338-49321
  Address = /139.63.20.119:49321
Handshake exception on socket: 49338
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 49321
New client attempting to connect on port: 49339-49321
  Address = /139.63.20.119:49321
This client on port 49339 has uuid 0810207d-f257-4faf-8f8e-818ea88ba1fc configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-8aa67c91-fdc7-4c65-a51a-bba36c82fd85, play-fork-run-0810207d-f257-4faf-8f8e-818ea88ba1fc
Taking next connection to: 49321
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49321
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 49321
